test/ProductTest.java

```diff
@@ -0,0 +1,51 @@
+import org.junit.Before;
```

Comment:
You only pushed the channel file. Please also push the product (add to git, commit and push)